### ISSUE TITLE
[Bug]: Program crashes if excluded username has emoji

### ISSUE THREAD

**Issue by Trainmaster2** [ThioJoe/YT-Spammer-Purge#347]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I was just plugging in random channels and scanning recent videos. I tried to exclude someone from a search and the program crashed. Turned out they had an emoji in their username.

### Release version

2.8.1

### Steps to reproduce

Try to exclude someone who has an emoji in their username.

### What platform are you seeing this problem on?

Windows

### Relevant log output

```shell
Input: exclude 14, 15
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 3726, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 3648, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 976, in exclude_authors
  File "encodings\cp1252.py", line 19, in encode
UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f633' in position 74: character maps to <undefined>
------------------------------------------------
Error Message: 'charmap' codec can't encode character '\U0001f633' in position 74: character maps to <undefined>

Unknown Error - Code: Z-3 occurred. If this keeps happening, consider posting a bug report
on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report
```

**Comment by ThioJoe** [comment]
Ah yup i know what the problem is, I'll fix it

**Comment by ThioJoe** [comment]
Should be fixed now, will be included when I make the patch release 2.8.2 soon.
